### ISSUE TITLE
Support PWA "Add to home screen"

### ISSUE THREAD

**Issue by anthonygraignic** [MetaMask/metamask-mobile#1966]
**Describe the usability problem**
MetaMask mobile has its own in-app web browser for Dapps, bringing a lot of cool features ! However I cannot add a shortcut to my home screen for PWA websites (only add them in my favorites inside the app).
I would be easier to embrace the "Add to Home screen" features on mobile so Dapp can be easily accessed from Home screen as regular mobile apps.

[Mozilla Web docs about "Add to Home screen"](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Add_to_home_screen)

**Screenshots**

|MetaMask Mobile browser| Brave Browser|
|---|---|
|![device-2020-11-09-111125](https://user-images.githubusercontent.com/12660783/98522470-683cbf00-227d-11eb-85f7-f75a4c22e371.png) | ![device-2020-11-09-111716](https://user-images.githubusercontent.com/12660783/98522469-65da6500-227d-11eb-87ba-227de34d59f7.png)|

**Expected behavior**
When visiting a PWA-enabled website e.g. https://mdn.github.io/pwa-examples/a2hs/, I should be able to add it to my home screen. (in example with the "Add to home screen" button on top left corner)

------------------------------------------------------------------------------------------------------
_filled out after submission_

**User impact | Usage frequency | Feasibility scores**
_How important is this to the user, what the user wants to accomplish | Daily, weekly, monthly | Small, medium, large_

**Comment by seiyria** [comment]
We're looking to make PWAs, and our users are primarily metamask mobile users when they use mobile. A feature like this would be great, especially if we could somehow seamlessly integrate the two somehow, so launching the PWA would launch metamask browser, but invisibly (like it does already). Is that something that'd be possible?

**Comment by Feloot** [comment]
Hello, any news on this ? We just launched a PWA for a blockchain service and we'd really like to take advantage of this.

**Comment by n3m** [comment]
Bumping this **_feature request_** again, since it would be a great addition to the MM Mobile Ecosystem for better adoption.

I'll try to get achieve a solution too in my own fork, but I'll probably fail. If I come up with something, I'll update here and create a PR

**Comment by fr0mpy** [comment]
bump

**Comment by Pirgo** [comment]
Bump

**Comment by playground** [comment]
Bump, 
Also, when detects a new version available, click ok to update breaks the app.

**Comment by jborlinic-a** [comment]
Yes please, this is of vital importance. The only real way to increase the usage of web3 dapps on mobile phones!

**Comment by SamAlchemiLab** [comment]
A PWA with metamask browser? I really could use it.

**Comment by Sgoldik** [comment]
Bump

**Comment by Jropp** [comment]
Bump
